js/exercise-03.js

```diff
@@ -0,0 +1,12 @@
+ 
+    /*1. Define function 'getTypeOf', which receives a value as paramenter and returns the type of this value
+        Use typeof to ckech the types for each of the main 5 primitive types (string, number, boolean, null, undefined)*/
+        function getTypeOf(value) {
```

Comment:
Unnecessary empty spaces in line

```suggestion
function getTypeOf(value) {
```